addons/repair/functions/fnc_addRepairActions.sqf

```diff
@@ -66,6 +66,8 @@ private _icon = QPATHTOF(ui\repair_0_ca.paa);
         _statement = {[_this select 1, _this select 0, _this select 2 select 0, "ReplaceWheel"] call DFUNC(repair)};
         _action = [_name, _text, _icon, _statement, _condition, {}, [_hitpoint], _position, 2] call EFUNC(interact_menu,createAction);
         [_type, 0, [], _action] call EFUNC(interact_menu,addActionToClass);
+
+        _processedSelections pushBack _selection;
```

Comment:
Now we do it twice for some selections.

Reply:
without this line double wheel hitpoints (with same selection) get 2 actions with same position
![4](https://user-images.githubusercontent.com/3867143/37853548-495ebf32-2ef8-11e8-9e96-c9a08ab2d2a8.png)